core-primitives/substrate-sgx/externalities/src/bypass.rs

```diff
@@ -0,0 +1,43 @@
+//! Converts maps to vecs for serialization.
+//! from https://github.com/DenisKolodin/vectorize
+//!
+//! `bypass` is necessary to force deriving serialization of complex type specs.
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+#[allow(unused)]
```

Comment:
Can you elaborate on that allow unused?

Reply:
That is just moved code, nothing new. My guess is that it's a sgx/std feature effect, where in one case we don't need it, but clippy would complain.

Reply:
Yeah probably, we can fix that some other time then 👍